style.css

```diff
@@ -643,7 +663,8 @@
   .modal-container {
     display: grid;
     grid-template-rows: repeat(8, auto);
-    padding: 5vw;
+    padding: 5vw 5vw 5vw 5vw;
+    height: 100vh;
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. The popup window in your mobile version is not rendering all the buttons. Kindly fix this :pray: :pray: 

Your design
![image](https://user-images.githubusercontent.com/98363075/202824276-e3f06c82-eb2f-4c5e-8382-c4c37ec12837.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/202824312-9e94b218-10be-4cbc-a67c-d226d37c1123.png)